testing/test_skipping.py

```diff
@@ -576,7 +576,7 @@ def test_strict_and_skip(self, testdir):
             def test_hello():
                 pass
         """)
-        result = testdir.runpytest("-rs --strict")
```

Comment:
Removed `--strict` from this test just because `--strict` doesn't really have anything to do with it.

Reply:
I just stumbled upon this test while working on #8384. It seems to me that with this change, the test (`test_strict_and_skip`) doesn't test what it intended to test, and it seems equal to `test_skip_no_reason` now.

Looking at 1fbd19b, the intention of the test was to check whether the `skip` marker is correctly registered by pytest, so I think this should be readded as `--strict-markers`, no?

Reply:
Good catch, thanks!

If you are working on #8384, would you mind doing that in the same PR?